packages/utils/src/browser/unsafeEvalSupported.js

```diff
@@ -0,0 +1,33 @@
+// Cache the result to prevent running this over and over
```

Comment:
Looks like a dupe of the same file above, should we be using the one in the utils package or the one in the core package?